### ISSUE TITLE
Correct speech output stops when making a range selection over lines or slurs

### ISSUE THREAD

**Issue by jmwmusic** [musescore/MuseScore#30465]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

When making a range selection if you try to select over a slur, hairpin, pedal marks (probably more) you start to hear “add next chord to selection” with each press and stop hearing the beats, sometimes if it comes across the end of the line (slur, pedal, etc) it kicks in again with the full announcement of the range selection, but not sure this is a consistent pattern.

Steps to Reproduce:
1.	 Fill a bar with various quavers.
2.	Put focus on beat 1, hold down shift and arrow along.  Notice it correctly announces each beat as you select and when you pause it announces the full range selection.  Since 4.6 it has now started saying “stem” in front of each bar/beat reference, it would be much better if this just said the beat and only the bar when it changes and not “stem”. 
3.	Add a hairpin from beats 2 to 3.5 (across 4 quavers).
4.	Put focus at the start of the bar and hold down shift while you arrow along the notes to select them. Notice that it announces the first two notes as above but once it hits the hairpin it announces “add next note to selection” with each arrow press.  It also does not announce the full range selection when you pause or finish selecting and so the user has to try and count how many notes forward they have arrowed, very difficult over long passages.

### Supporting files, videos and screenshots

N/A

### In which versions of MuseScore Studio is this issue present?

4.6.1

### Regression

I was unable to check

### Operating system

Windows 11 with NVDA

### Checklist

- [x] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [x] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [x] I have attached all requested files and information to this report
- [x] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible
